### ISSUE TITLE
Swap back to the metaflow security group

### ISSUE THREAD

**Issue by jaklinger** [nestauk/daps_utils#74]
1. Set up the luigi server in the metaflow security group
2. Set up the database in the metaflow security group
3. Swap the batch compute environment back to the metaflow security group

A subsequent issue should be raised for doing steps 1) and 2) using terraform alongside the metaflow cloudformation

**Comment by jaklinger** [comment]
Linked to https://github.com/nestauk/research_daps/issues/33

**Comment by bishax** [comment]
Moving DB and luigi server into metaflow security group seems simpler (and more secure?) than the reverse which I suggested in nestauk/research_daps#33

**Comment by jaklinger** [comment]
Indeed, though in practice I'll need to setup a new DB and luigi server, that's the faffy bit, but only an hours work
